libs/perun/utils/src/lib/perun-utils.ts

```diff
@@ -864,3 +864,10 @@ export function isMemberIndirectString(member: RichMember): string {
   }
   return 'UNKNOWN';
 }
+
+/**
+ * Return true, if GUI is run locally, false otherwise.
+ */
+export function isRunningLocally(): boolean {
+  return location.hostname === 'localhost' || location.hostname === '127.0.0.1';
```

Comment:
The hostname could differ when running local GUI and [rerouting localhost](https://perunaai.atlassian.net/wiki/spaces/STRIBOG/pages/66879489/Https+localhost) to enable https. Can this be an issue?

Reply:
In this case, we want to pretend non-localhost access and the GUI should behave like it is not the localhost, so this shouldn't be a problem. :no_good_man: